CMakeLists.txt

```diff
@@ -690,14 +693,10 @@ set(IREE_PUBLIC_INCLUDE_DIRS "${IREE_COMMON_INCLUDE_DIRS}"
 add_subdirectory(build_tools/benchmarks)
 
 #-------------------------------------------------------------------------------
-# Transitional directories.
+# Samples
 #-------------------------------------------------------------------------------
 
-# The runtime uses tools and targets from elsewhere in the tree in a way
-# that is not supported by CMake, so we include it last (it aggregates
-# many things into top-level APIs). This should resolve once we finish
-# relayering everything.
-add_subdirectory(runtime)
```

Comment:
Hm.. need to look a bit more closely at this for the `runtime/bindings/python/` package:

https://github.com/google/iree/blob/f56b906312b1b621ec86e3c96aabddd2325a0f40/runtime/bindings/python/CMakeLists.txt#L196-L204

```
CMake Error at runtime/bindings/python/CMakeLists.txt:196 (install):
  install TARGETS given target "iree_tools_iree-benchmark-module" which does
  not exist.
```

https://source.cloud.google.com/results/invocations/784aeae7-983e-4832-b10b-416aed15da9f/targets/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Fx86-swiftshader%2Fpresubmit/log

Reply:
That will need a bit more work to move around. I just reverted my changes to subdirectory include order to fix it.